capabilities.js

```diff
@@ -4,6 +4,19 @@ define('capabilities', ['settings'], function(settings) {
         return !!m && m.matches;
     }
 
+    function detectOS() {
```

Comment:
it's odd that it's called `detectOS` but this works for extracting only Mac OS X version numbers